### ISSUE TITLE
[feature] nice to have ideas for future update

### ISSUE THREAD

**Issue by guilders** [recalbox/recalbox-os#885]
Hello All,

Here are some ideas for a future update of the V4 version :

-be able to pause in game

-a "global favourite" menu directly available from the ES menu, next to the different emulators names and that would list all games flagged as favourite from all the systems. So one place to go for all favourite games

-possibility to sort games by editor (ex: I'd like to see in MAME all the Capcom games) or genre (ex: I'd like to see all shoo't em up  existing to try a new one)

-have the game pictures showed in higher resolution. Currently it's very small and not easy to really see what the game looks like (or be able to choose between small, medium, large)

-PS4 pads compatibility in Bluetooth

-if playable with a Pi 3, look at adding new systems, I think of Naomi, Taito type X, ZN

-where are you with the implementation of scrapped images collection ? This would be nice to have, even though if I prefer an in-game picture rather than the cover.

Anyway, wonderful job and I just proceeded with a small donation to support your work :)

**Comment by daaa57150** [comment]
Hi,

- I'm pausing in game by opening the retroarch menu (hotkey + B I think)
- The favourites menu does exist already, maybe it's only in the V4, th'at what I have and it's there. It's my 1st recalbox I don't know about the previous versions

**Comment by guilders** [comment]
Thanks for the in-game pause, I will try.

Regarding the favourites, do you have a Favourite menu on the console selection screen? So next to MAME, FBA, SNES, NES, ... ? Or do you have a Favourite menu in each different emulator?

**Comment by digitalLumberjack** [comment]
And the game pictures are showed in higher resolution with the new theme.
Did you tried the 4.0 @guilders ?

**Comment by guilders** [comment]
yes I am on the latest beta version and I tried yesterday the new recalbox theme.
I like better the layout and the pictures, now showing on the right side, also looked bigger. This is nice.

**Comment by daaa57150** [comment]
If you're on 4.0 then you should have the favourites menu:
- An entry next to all emulators
- In each different emulator the favourites are marked with a star and are put on the top

Highlight a game in any emulator and press Y I think, it's indicated on the bottom left of the screen, and you should see what I'm talking about

**Comment by guilders** [comment]
Thanks, great, I will give a try this afternoon :)

**Comment by guilders** [comment]
I tried the favourites and it is working as expected, except it is not saved, so lost at each reboot.

With the new layout, pictures are indeed bigger, now it looks just great.

Also tried the in-game pause, using hotkey+B, also working as expected, fine.

**Comment by digitalLumberjack** [comment]
You must shutdown cleanly to have your modifications saved. START + QUIT

**Comment by guilders** [comment]
ok, thanks
